src/components/common/Footer/Footer.tsx

```diff
@@ -0,0 +1,28 @@
+import classame from "classnames";
+import Image from 'next/image'
+import { AiOutlineTwitter,AiOutlineInstagram } from 'react-icons/ai'
+
```

Comment:
使ってないライブラリは削除しよう！
（後でEslint設定します）

```suggestion
import Image from 'next/image'
import { AiOutlineTwitter } from 'react-icons/ai'
```